README.ja.md

```diff
@@ -238,7 +238,7 @@ Starship の設定方法の詳細に関しては、[ドキュメント](https://
 
 このプロジェクトは [all-contributors](https://github.com/all-contributors/all-contributors) の仕様に従っています。 どんな種類の貢献でもお待ちしています！
 
-## 影響を受けたプロダクト
+## 💭影響を受けたプロダクト
```

Comment:
<img width="206" alt="Screen Shot 2019-09-28 at 20 48 12" src="https://user-images.githubusercontent.com/12455284/65815989-5c389e00-e231-11e9-9746-f1cfea6e83e4.png">

It seems that  "💭" was deleted by mistake.